plugin/plugins/bilibili_danmaku/danmaku_core.py

```
@@ -282,8 +315,16 @@ async def _fetch_buvid3(self) -> str:
             self._log(f"获取临时 buvid3 失败: {e}", "warning")
         return ""
 
-    async def _get_danmaku_server_info(self, real_room_id: int) -> tuple[str, str]:
-        """获取弹幕服务器地址和 token（带 WBI 签名）"""
+    async def _get_danmaku_server_info(self, real_room_id: int) -> tuple[list, str]:
+        """
+        获取所有弹幕服务器地址列表和 token（带 WBI 签名）。
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)